test/unit/alignment/matrix/detail/affine_cell_proxy_test.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include "gtest/gtest.h"
 
+#include <seqan3/test/expect_same_type.hpp>
```

Comment:
Alphabetical order and without a line in between?

Reply:
To be honest, we don't have an explicit rule, yet. We have some tests that split test includes from library includes and some not. So it's a good question. I change it to alphabetically order.